src/supplemental/nanolib/ringbuffer/ringbuffer.c

```diff
@@ -695,6 +695,18 @@ static int put_msgs_to_aio(ringBuffer_t *rb, nng_aio *aio)
 	return 0;
 }
 
+int ringBuffer_fullOp(ringBuffer_t *rb, enum fullOption fullOp)
```

Comment:
How about renaming to ringBuffer_set_fullOp

Reply:
you are right, rename to ringBuffer_set_fullOp